packages/stems/src/components/Button/Button.module.css

```diff
@@ -95,19 +108,25 @@
   font-weight: var(--font-bold);
 }
 
-.primary .icon g path,
+.primary .icon path,
```

Comment:
I think this was originally an oversight. Not that due to descendant selector behavior, this will still match elements which are nested under a `g`. But it was failing to correctly set the fill on some icons which do _not_ contain a `g` element.